### PR TITLE
chore(DEVHAS-683): Delete HAS SLOs

### PR DESCRIPTION
**Pull request by johnmcollier**
- Removes the HAS SLO alerts
- Removes the tests for the removed alerts
- Removes the SLO dashboard widgets for HAS

### CODE REVIEW

**Comment by johnmcollier** [comment]
I also checked the availability metrics dashboard and didn't see anything specific to HAS to remove

**Review by yftacherzog — APPROVED** [review]
lgtm

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml [hunk 1]
```diff
@@ -2047,16 +1561,15 @@ data:
...
       "tags": [],
       "templating": {
         "list": [
           {
             "current": {
-              "selected": true,
+              "selected": false,
               "text": "rhtap-observatorium-production",
-              "value": "rhtap-observatorium-production"
+              "value": "P22466E8E7855F1E0"
```
Comment (amisstea) [inline dashboards/grafana-dashboard-rhtap-slos.configmap.yaml:1572]: This change looks unintentional. Please ensure the prod datasource is selected by default.

**Review by amisstea — CHANGES_REQUESTED** [review]

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml (hunk 1 above)
Comment (johnmcollier) [reply dashboards/grafana-dashboard-rhtap-slos.configmap.yaml]: I'm not sure why that's happening, it seems that Grafana's doing that automatically whenever I export without even touching the datasource.

Anyhow, I've reverted the change

**Review by amisstea — APPROVED** [review]
/lgtm

**Pull request merged**